src/Admin/AdherentAdmin.php

```diff
@@ -3,22 +3,17 @@
 namespace AppBundle\Admin;
 
 use AppBundle\Entity\Adherent;
-use AppBundle\Form\ActivityPositionType;
-use AppBundle\Form\GenderType;
+use AppBundle\Form\{ActivityPositionType, GenderType};
 use AppBundle\Intl\UnitedNationsBundle;
 use Misd\PhoneNumberBundle\Form\Type\PhoneNumberType;
-use Sonata\AdminBundle\Admin\AbstractAdmin;
-use Sonata\AdminBundle\Datagrid\ListMapper;
-use Sonata\AdminBundle\Datagrid\DatagridMapper;
-use Sonata\AdminBundle\Form\FormMapper;
-use Sonata\AdminBundle\Show\ShowMapper;
+use Sonata\AdminBundle\{
+    Admin\AbstractAdmin, Datagrid\ListMapper, Datagrid\DatagridMapper, Form\FormMapper, Show\ShowMapper
+};
```

Comment:
You should use one use statement per class.
If not we won't be able to use `grep` anymore, for example.

http://www.php-fig.org/psr/psr-2/#namespace-and-use-declarations

> There MUST be one use keyword per declaration.